common/on-chip-calib.cpp

```diff
@@ -901,6 +903,7 @@ namespace rs2
     {
         try
         {
+            std::this_thread::sleep_for(std::chrono::milliseconds(1000)); // W/A that allows for USB2 exposure to settle
```

Comment:
Please confirm that there is no other way to solve this (like some trigger from the exposure control)

Reply:
The other way is to generate a mechanism that will ensure the camera's exposure is stabilized. But since we can't know it from metadata (the exposure value reported is always the same, but the picture itself is changing), it must be done using Computer Vision methods, which is quite complicated.
In the tests, having 500 msec delay was sufficient to make the algorithm converge in more than 90% of cases. So making it 1000 is has some spair time to make sure it will also run smoothly in low-end platforms

Reply:
ok